README.md

```diff
@@ -219,73 +128,16 @@ bug and test the latest stable version.
-unpacking debuginfo...
-hotspot-debuginfo-v1.3.0-391-g590f810/usr/bin/hotspot
-hotspot-debuginfo-v1.3.0-391-g590f810/usr/lib64/libexec/hotspot-perfparser
-starting debuginfod in docker...
-to use it on your host system, set:
-
-    export DEBUGINFOD_URLS="127.0.0.1:12345 https://debuginfod.centos.org/ https://debuginfod.archlinux.org"
-
-[Fri Dec  2 19:58:56 2022] (16/16): opened database /opt/app-root/src/.debuginfod.sqlite
-[Fri Dec  2 19:58:56 2022] (16/16): sqlite version 3.7.17
-[Fri Dec  2 19:58:56 2022] (16/16): started http server on IPv4 IPv6 port=12345
-...
-```
-
-Keep that `debuginfod` process running. You can now start using the local debuginfod in combination with
-the official Centos instance to get debug information for all the dependencies and build artifacts of hotspot.
-Here is one example for that using GDB:
+## Building Hotspot
 
-```bash
-$ ./hotspot-v1.3.0-391-g590f810-x86_64.AppImage &
-$ export DEBUGINFOD_URLS="127.0.0.1:12345 https://debuginfod.centos.org/ https://debuginfod.archlinux.org"
-$ gdb -p $(pidof hotspot)
-...
-This GDB supports auto-downloading debuginfo from the following URLs:
-127.0.0.1:12345 https://debuginfod.centos.org/ https://debuginfod.archlinux.org
-Enable debuginfod for this session? (y or [n]) y
-Debuginfod has been enabled.
-To make this setting permanent, add 'set debuginfod enabled on' to .gdbinit.
-Downloading 37.95 MB separate debug info for /home/milian/Downloads/squashfs-root/usr/bin/hotspot
-...
-(gdb) bt
-#0  0x00007f70bd5140bf in __GI___poll (fds=0x2c83f40, nfds=5, timeout=0) at ../sysdeps/unix/sysv/linux/poll.c:29
-#1  0x00007f70b7a4c3fc in g_main_context_iterate.isra () from /tmp/.mount_hotspoBCRbef/usr/lib/libglib-2.0.so.0
-#2  0x00007f70b7a4c52c in g_main_context_iteration () from /tmp/.mount_hotspoBCRbef/usr/lib/libglib-2.0.so.0
-#3  0x00007f70bdd08aa4 in QEventDispatcherGlib::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) ()
-   from /tmp/.mount_hotspoBCRbef/usr/lib/libQt5Core.so.5
-#4  0x00007f70bdcb361b in QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) ()
-   from /tmp/.mount_hotspoBCRbef/usr/lib/libQt5Core.so.5
-#5  0x00007f70bdcbb26c in QCoreApplication::exec() () from /tmp/.mount_hotspoBCRbef/usr/lib/libQt5Core.so.5
-#6  0x000000000043a23e in main (argc=<optimized out>, argv=<optimized out>) at /__w/hotspot/hotspot/src/main.cpp:194
-```
+Building hotspot from source gives you the latest and greatest, but you'll have to make sure all its
+dependencies are available. Most users should probably [install hotspot](#getting-hotspot) from the distro
+package manager or as an [AppImage](#for-any-linux-distro-appimage).
 
-Note that source files are not supported in this way yet, as that would blow up the size of the docker image
-and thereby slow down our CI.
+For everyone that wants to contribute to Hotspot or use the newest version without the Appimage detailed
+notes are found at [HACKING](HACKING#table-of-contents).
```

Comment:
dito, `(HACKING.md#table-of-contents)`

Reply:
Checked, with GitHub and vscode markdown editing both work (you can "click" in the preview and on the page; language-server suggest the contained anchors if you type `[README#`.

Just specify if you want that changed and I'll do push that.

Any other changes needed / suggested?

Reply:
if it works then I'm fine with that then

odd that it doesn't need the file extension though :shrug: